src/js/index.js

```diff
@@ -0,0 +1,89 @@
+const totalDisplay = document.getElementById('total')
+const numbers = document.querySelectorAll('.digit')
+const operations = document.querySelectorAll('.operation')
+const modifiers = document.querySelectorAll('.modifier')
+
+let leftNum = '0'
+let rightNum = ''
+let oper = ''
+
+function updateTotalDisplay() {
+  totalDisplay.innerText = leftNum + ' ' + oper + ' ' + rightNum
+}
+
+function pressNumber(number) {
+  console.log(number)
+  if (oper === ''){
+    leftNum = leftNum === '0' ? number : Number(leftNum + number)**2 >= 1000000 ? leftNum : Number(leftNum + number).toString()
+  }
+  else {
+    rightNum = rightNum === '0' ? number : Number(rightNum + number)**2 >= 1000000 ? rightNum : Number(rightNum + number).toString()
+  }
+  updateTotalDisplay()
+}
+
+function pressOperation(operation) {
+  if (operation === '=') {
+    calculate()
+    return
+  } else if (rightNum === '') {
+    oper = operation
+    updateTotalDisplay()
+  } else {
+    calculate()
+    oper = operation
+    totalDisplay.innerText = leftNum + ' ' + oper
```

Comment:
```suggestion
    updateTotalDisplay()
```
이 부분은 `updateTotalDisplay()` 함수와 동일한 역할을 하는 것으로 보여서 대체해도 될 것 같습니다~!